webpack.prod.js

```diff
@@ -27,8 +27,11 @@ module.exports = {
     }),
     new MiniCssExtractPlugin({ filename: "styles.css" }),
     new webpack.DefinePlugin({
-      "process.env.KINTO_ADMIN_VERSION": JSON.stringify(version),
-      "process.env.ASSET_PATH": JSON.stringify(ASSET_PATH),
+      "process.env": {
+        ASSET_PATH: JSON.stringify(ASSET_PATH),
```

Comment:
This maybe deservers a mention in the README?

Reply:
I think it does too. I was going to take care of that in a follow-up PR because I think a good section of the documentation will have to change if we're going to encourage downloading a release a building from source rather than `npm install` ing the `KintoAdmin` component.